api/proto/solaris/v1/solaris.proto

```diff
@@ -1,5 +1,116 @@
 syntax = "proto3";
 
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
 package solaris.v1;
 option go_package = "./solaris/v1;solaris";
 
+// Service exposes the public interface for the Solaris DB
+service Service {
+  // CreateLog creates then new log
+  rpc CreateLog(Log) returns (google.protobuf.Empty);
+  // UpdateLog changes the log settings (tags)
+  rpc UpdateLog(Log) returns (Log);
+  // QueryLogs requests list of logs by the query request ordered by the log IDs ascending order
+  rpc QueryLogs(QueryLogsRequest) returns (QueryLogsResult);
+  // DeleteLogs removes one or more logs
+  rpc DeleteLogs(DeleteLogsRequest) returns (DeleteLogsResult);
+  // AppendRecords appends a bunch of records to the log
+  rpc AppendRecords(AppendRecordsRequest) returns (Log);
+  // QueryRecords read records from one or many logs, merging them together into the result set
+  // sorted in ascending or descending order by the records IDs (timestamps)
+  rpc QueryRecords(QueryRecordsRequest) returns (QueryRecordsResult);
+}
+
+// Record represents one record of a log
+message Record {
+  // id is the record unique identifier. A record ID is ULID, so it is soreted and globaly unique.
```

Comment:
nit: Not sure if we need to say that it is ULID specifically

Reply:
well, ok, will rephrase